autotime/__init__.py

```diff
@@ -5,11 +5,10 @@
 
 from IPython.core.magics.execution import _format_time as format_delta
 
-if sys.platform == 'win32':
-    if sys.version_info >= (3, 3):
-        _timer = time.perf_counter
-    else:
-        _timer = time.clock
+if sys.version_info >= (3, 3):
+    _timer = time.perf_counter
+elif sys.platform == 'win32':
```

Comment:
I don't think you need to check platform _at all_. `perf_counter` is available on windows.

Reply:
`perf_counter` is introduced from Python 3.3
On Python 2.7, 'clock' is preferred on windows.

Reply:
Note the condition flow is changed, `perf_counted` is always selected for Python 3.3+ and `clock/time` is used otherwise depending on the platform.